ui/webui/src/components/installation/ReviewConfiguration.jsx

```diff
@@ -14,16 +14,143 @@
  * You should have received a copy of the GNU Lesser General Public License
  * along with This program; If not, see <http://www.gnu.org/licenses/>.
  */
-import React from "react";
+import cockpit from "cockpit";
+import React, { useEffect, useState } from "react";
 
 import {
-    Form
+    Checkbox, Title,
+    DataList, DataListItem,
+    DataListItemRow, DataListItemCells,
+    DataListCell,
+    DescriptionList, DescriptionListGroup,
+    DescriptionListTerm, DescriptionListDescription,
+    Flex, Popover,
 } from "@patternfly/react-core";
 
+import {
+    getSelectedDisks, getDeviceData,
+} from "../../apis/storage.js";
+
+import {
+    getLanguage, getLanguageData,
+} from "../../apis/localization.js";
+
+import HelpIcon from "@patternfly/react-icons/dist/esm/icons/help-icon";
+
+const _ = cockpit.gettext;
+
 export const ReviewConfiguration = () => {
+    const [systemLanguage, setSystemLanguage] = useState("YYY");
+    const [selectedDisks, setSelectedDisks] = useState();
+    const [deviceData, setDeviceData] = useState({});
+
+    useEffect(() => {
+        getLanguage()
+                .then(res => {
+                    console.info("language code");
+                    console.info({ res });
+                    getLanguageData({ lang: res }).then(res => {
+                        console.info("language data");
+                        console.info({ res });
+                        setSystemLanguage(res["native-name"].v);
+                    }, console.error);
+                }, console.error);
+    }, []);
+
+    useEffect(() => {
```

Comment:
You can use a single useEffect call. Just run both apis in the body of the single useEffect, one after the other.

Reply:
OK, changed. :)